fsevents.cc

```diff
@@ -14,13 +14,13 @@
 
 #include "src/storage.cc"
 namespace fse {
-  class FSEvents : public node::ObjectWrap {
+  class FSEvents : public Nan::ObjectWrap {
   public:
-    FSEvents(const char *path, Nan::Callback *handler);
+    FSEvents(const char *path);
     ~FSEvents();
 
     // locking.cc
-    bool lockStarted;
+    bool lockStarted = false;
```

Comment:
Can you do this in the constructor?  This is a C++11 construct and we (nominally) still support Node.js v0.10 and v0.12, which are C++03.